articles/extensions/authorization-extension/v2/implementation/setup.md

```diff
@@ -0,0 +1,150 @@
+
+You can then configure group mappings to look at a user's profile if they're connecting with the Active Directory connection. When the extension sees that the person is a part of the "Administrative" group, it will automatically make the user a member of your company's Admin group.
+
+![Group Mappings](/media/articles/extensions/authorization/group-mapping-v2.png)
+
+## Roles
+
+The roles that you will create will depend on the access to certain permissions in your application. For example, let's say that you have an application that allows employees to enter in company expenses. You want all employees to be able to submit expenses, but want certain Finance users to have more admin type of actions such as being able to approve or delete expenses. These actions can be mapped to [Permissions](#permissions) and then assigned to a certain Role.
+
+You can create different types of Roles such as: Expense Admins, Expense Manager, and Expense User for your Expense Management Tool.
+
+![Roles](/media/articles/extensions/authorization/roles.png)
+
+To add a role, click the **CREATE ROLE** button from the **Roles** section of the dashboard. Then choose the application this Role applies to (such as Expense Management Tool) and then add a name of the role (such as Expense Admins) and a description of the role. Then select the permissions you wish to grant to this role. If you haven't yet created your permissions you can add them later to an exisiting Role.
+
+![Add a New Role](/media/articles/extensions/authorization/add-role.png)
+
+Once you have a **Role** created, you can add it to a user so they can then have the associated **Permissions**. To add a role to a user, find the user in the **Users** section, then click the **Roles** tab. Then click **ADD ROLE TO USER** to choose which roles you wish to assign to a user, then click **SAVE**.
+
+![Add Role to User](/media/articles/extensions/authorization/add-role-to-user.png)
+
+## Permissions
+
+Permissions are the actions or functions that can be added to Roles.
+
+Using the previous example of an Expense application, let's look at possible roles and how they can be associated with certain permissions:
+
+- Role: Expense User
+  - Permissions:
+    - View their own expenses
+    - Add a new expense
+
+- Role: Expense Admin
+  - Permissions:
+    - Approve expenses
+    - View all user expenses
+    - Delete expenses
+    - Add a new expense
+
+To create a new permission, go to the **Permissions** section of the Authorization Extension dashboard.
+
+![Permissions](/media/articles/extensions/authorization/permissions.png)
+
+Then click the **CREATE PERMISSION** button. Then enter the name of the permission, the description and select the application for which this permission applies.
+
+![Create Permission](/media/articles/extensions/authorization/create-permission.png)
+
+Once you have your permissions created, you can associate them with [Roles](#roles).
+
+## Keep Reading
```

Comment:
Now that this page is not the last of the `Get Started` section, you probably want to move this paragraph at the `Set Up the Extension` https://auth0-docs-content-pr-5025.herokuapp.com/docs/extensions/authorization-extension/v2/implementation/configuration